ably/rest_client.go

```diff
@@ -92,27 +86,24 @@ func (c *RestClient) Stats(params *PaginateParams) (*PaginatedResource, error) {
 }
 
 func (c *RestClient) Get(path string, out interface{}) (*http.Response, error) {
-	req, err := http.NewRequest("GET", c.RestEndpoint+path, nil)
+	req, err := http.NewRequest("GET", c.options().restURL()+path, nil)
 	if err != nil {
 		return nil, err
 	}
 	req.Header.Set("Accept", "application/json")
-	req.SetBasicAuth(c.Auth.Options.Token, c.Auth.Options.Secret)
-	res, err := c.httpclient().Do(req)
-
+	// TODO(rjeczalik): add support for token auth
```

Comment:
This was actually the first thing I've noticed - I've collected a fair amount things that need either to be fixed, missing or polished a bit, I keep them in my trello board as I didn't want to create noise in github issues - https://trello.com/b/pHESio8l/ably-go. Or you feel you want to give feedback on those and I should move them to github isseus?